src/cli/borsh_io.rs

```diff
@@ -0,0 +1,88 @@
+use aurora_engine::{fungible_token::FungibleReferenceHash, admin_controlled::PausedMask};
+use aurora_engine_types::types::{Address, RawU256, NEP141Wei};
+use borsh::{BorshSerialize, BorshDeserialize};
+use near_primitives::types::{AccountId, Balance};
+use serde::{Serialize, Deserialize};
+
+
+#[derive(BorshSerialize)]
+pub struct GetStorageAtInput {
```

Comment:
Do we need to repeat these? Couldn't we import them from `aurora_engine::parameters` instead?

Reply:
I see aurora engine release 2.7.0 has not included args in `borsh_io.rs` yet. Is there a current tag that I can use in importing through `Cargo.toml` so that I can remove this file?

Reply:
Never mind, there are no more methods using this (`begin_chain`, `withdraw`)